docs/getting-started.md

```diff
@@ -59,12 +59,29 @@ scalar.cassy.server.slave_command_path=/path/to/cassy/build/install/cassy/bin
 # URI of a blob store to manage backup files 
 scalar.cassy.server.storage_base_uri=s3://your-bucket
 
+# Enum to tell Cassy which cloud service to use (aws_s3, azure_blob)
+scalar.cassy.server.storage_type=aws_s3
+
 # URL of JDBC for managing some metadata such as backup and restore histories
 scalar.cassy.server.metadata_db_url=jdbc:sqlite:cassy.db
 
 # URL of SRV record. It is used to know and pause Cassandra application nodes to take a cluster-wide snapshot. If you don't use the feature, it can be omitted or the value can be left blank.
 scalar.cassy.server.srv_service_url=_app._tcp.your-service.com
 ```
+### Cloud Storage Type Configuration
+To change the cloud service Cassy uses, the following properties of the configuration file will need to be modified:
+1. `storage_base_uri`: the url to your S3 bucket or Azure blob.
```

Comment:
```suggestion
1. `storage_base_uri`: the uri to your S3 bucket or Azure blob.
```
@yume-ind I'm just wondering, in Azure case, you said it should specify container name, but is there any reason why we can't specify the full URI ? `https://cassydev.blob.core.windows.net/scalar-test`
It just doesn't work ? 
Since it's currently named `storage_base_uri`, users will expect URI here, but if we need to specify a container name, it's a bit confusing for users, so we might need to rename it if we need to specify container name.

Reply:
There is a way. 
If  `storage_base_uri=https://cassydev.blob.core.windows.net/scalar-test`, we can update the code to extract the container name from this URI like the following.
1. Verify the blob storage address `storage_base_uri` is the same as the one from the connection string environment variable, i.e. `https://cassydev.blob.core.windows.net`
2. Extract the container name from the `storage_base_uri` via string manipulation.

What do you think ?

Reply:
@Torch3333 If we store the URI in the config, what is the point of having it in an environment variable ? 
Was there any particular reason of having `https://cassydev.blob.core.windows.net` in an env and `scalar-test` in the config separately ?

Reply:
We choose to use the `connection string` to login (and which is passed as an env variable) because it seems the easiest way to set up authentication credentials for the blob. 
The `connection string` can be directly generated on the blob page. For example, for the `cassydev` blob, this will be here
https://portal.azure.com/#@scalar-labs.com/resource/subscriptions/b647bf0e-0566-4b28-ac8c-37b24f3b7061/resourceGroups/indetail/providers/Microsoft.Storage/storageAccounts/cassydev/keys
But the `connection string` only contains the credentials for a specific blob but does not point to a container of this blob, hence the user need to provide the `container` name separately that will be used to store the Cassandra snapshot. 
That's why, we used the `storage_base_uri` to pass the `container` name.
Asking the user to set these 2 parameters separately seemed to be the best compromise between usability and configurability.

Reply:
OK, thank you for the detail explanation.
Let me make sure one thing. 
So, at the moment, we set `connection string` in an env manually, right ?

I think it would be a bit better for users to specify with URI format like this `storage_base_uri=https://cassydev.blob.core.windows.net/scalar-test`.

Reply:
```
So, at the moment, we set connection string in an env manually, right ?
```
Yes, the `connection_string` has to be set manually. 

```
I think it would be a bit better for users to specify with URI format like this storage_base_uri=https://cassydev.blob.core.windows.net/scalar-test.
```
Should I implement it as explained in this message ?
https://github.com/scalar-labs/cassy/pull/36#discussion_r399918842

Reply:
Yes, please. it would be great if it is done in another PR.

Reply:
Ok